eng/common/pipelines/templates/steps/update-docsms-metadata.yml

```diff
@@ -22,6 +22,12 @@ parameters:
   - name: DailyDocsBuild
     type: boolean
     default: false
+  - name: CodeOwners
+    type: string
+    default: ""
+  - name: DefaultCodeOwnerFile
```

Comment:
Lets not add this default code owner file. We should just fix the tool to handle the fallback.